Assets/BossRoom/Scripts/Shared/Game/UI/UIHealth.cs

```diff
@@ -21,13 +21,16 @@ public void Initialize(NetworkVariableInt networkedHealth, int maxValue)
             m_HitPointsSlider.minValue = 0;
             m_HitPointsSlider.maxValue = maxValue;
             m_HitPointsSlider.value = networkedHealth.Value;
+            // disable slider when we're at full health!
+            m_HitPointsSlider.gameObject.SetActive(m_HitPointsSlider.value != m_HitPointsSlider.maxValue);
```

Comment:
You could just remove these three lines and call `HealthChanged` directly with `networkedHealth.Value` as `newValue`, and move the comment as well. Might be worth adding a public toggle here as well to have the disabling when at full health optional maybe? Seems odd to have players' health bars disabled at start but that's just me. I'm good either way.

Reply:
done.